queue_services/entity-emailer/src/entity_emailer/email_processors/filing_notification.py

```diff
@@ -97,15 +97,16 @@ def _get_pdfs(
         else:
             corp_name = business.get('legalName')
 
+        # business_data won't be available for incorporationApplication
         business_data = Business.find_by_internal_id(filing.business_id)
         receipt = requests.post(
             f'{current_app.config.get("PAY_API_URL")}/{filing.payment_token}/receipts',
             json={
                 'corpName': corp_name,
                 'filingDateTime': filing_date_time,
-                'effectiveDateTime': effective_date,
+                'effectiveDateTime': effective_date if effective_date != filing_date_time else '',
```

Comment:
is this to ensure that if the filer hasn't processed the filing yet that an empty string will be displayed on the IA where recognition date and time are currently displayed?

Reply:
effective_date and filing_date_time will be different only when a filing is in future date. When I check the filing-ui it is not passing effective_date to download receipt

Reply:
oh, this is for the receipt.  missed that.